gbif-occurrence-linkage/main.py

```diff
@@ -152,16 +130,17 @@ def run_local(example: str) -> None:
     Will call the DiSSCo API to retrieve the specimen data.
     A record ID will be created but can only be used for testing.
     :param example: The full URL of the Digital Specimen to the API (for example
-    https://dev.dissco.tech/api/v1/specimens/TEST/M88-SJK-MDP
+    https://dev.dissco.tech/api/v1/digital-specimen/TEST/TYB-XNH-53H
```

Comment:
This needs to become a 'real' example I guess when we have usable DOIs

Reply:
this would only work in the dev env anyway because of the data model changes

Reply:
At the moment yes, but when it deployed again